ehrql/backends/tpp.py

```diff
@@ -218,10 +218,18 @@ def medications(self):
         """
             SELECT DISTINCT
                 Appointment_ID AS appointment_id,
+                CAST (BookedDate AS date) AS booked_date,
+                CAST(StartDate AS date) AS start_date,
                 Patient_ID AS patient_id,
-                CAST(BookedDate AS date) AS booked_date,
-                CAST(StartDate AS date) AS start_date
-            FROM Appointment
+                Description AS status
+            FROM Appointment AS appt
+            LEFT JOIN (
+                SELECT
+                    Code,
+                    Description
+                FROM DataDictionary
+                    WHERE [Table] = 'Appointment') AS dict
```

Comment:
I'd suggest indenting `WHERE` to the same level as `FROM` and terminating the sub-query on a new line

```suggestion
                WHERE [Table] = 'Appointment'
            ) AS dict
```